app/controllers/mixins/actions/vm_actions/resize.rb

```diff
@@ -52,7 +52,11 @@ def resize_vm
                 flavor_id = params['flavor_id']
                 flavor = find_record_with_rbac(Flavor, flavor_id)
                 old_flavor_name = @record.flavor.try(:name) || _("unknown")
-                @record.resize_queue(session[:userid], flavor)
+                # TODO: still need to determine whether the next line should be deleted or replaced
```

Comment:
So I think this is the one remaining issue to work through for this PR. I'm not quite sure how to handle request ID for the edit/approval case.

Reply:
In any case I"m digging into the reconfigure code to compare it -- looks like right now what's being passed around is completely different between the two cases.

Reply:
It looks like the best thing to do here is to change the existing resize UI code to correspond to what the reconfigure code does to pass IDs around. The existing UI code that I didn't need to mess with for this PR doesn't pass along the request ID.